pos_self_service_weighing_base/static/src/xml/Chrome.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<templates id="template" xml:space="preserve">
+
+    <t
+        t-name="Chrome"
+        t-inherit="point_of_sale.Chrome"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath expr="//TicketButton" position="attributes">
+            <attribute name="t-if">showTicketButton()</attribute>
+        </xpath>
+    </t>
+
```

Comment:
I talked with @robinkeunen after forgetting to read his comment from earlier today → hiding the close button would also be a satisfactory solution. This would also allow getting rid of some code elsewhere related to the close pop-up.

Reply:
done